includes/lemin.h

```diff
@@ -88,7 +92,7 @@ typedef struct			s_lines
 
 typedef struct			s_ant_farm
 {
-	size_t				ants;
+	long long			ants;
```

Comment:
good call otherwise we would have a problem checking the valid ants on max int value

Reply:
exactly